articles/tutorials/data-scenarios/index.md

```diff
@@ -0,0 +1,257 @@
+
+
+## Why not put all the app’s data in the Auth0 data store?
+
+The Auth0 data store is highly specialized for storing authentication data. Storing any authentication-related data beyond the default user information in the Auth0 data store should only be done in specific cases. Here are some reasons why you should not use the Auth0 data store when you don't have to:
+
+### Scalability
+
+It is best not to store non-authentication-related data in the Auth0 data store because the Auth0 data store has limited scalability and your app’s data could exceed that limit. Using an external database allows you to keep things simple on the Auth0 side, to be able to efficiently manage your user authentication, while leaving the heavy database lifting up to a separate database formatted to efficiently store your extra data. 
+
+### Performance
+
+Beyond scale, keeping your authentication data separate from any other data in your app is important because the two sets of data are also likely accessed with different frequencies. The Auth0 data store is not optimized to be queried with extremely high frequency. It is better to leave this up to a specialized database service, optimized for fast, large data operations.
+
+### Flexibility 
+
+By using a separate database, your access to your users' authentication data is concentrated in a small chunk of code, and your access to all other data is grouped separately. Your organizational demands may not be met by the Auth0 data store because it is built to accomodate only the user profiles and their Metadata. Certain actions that you might require from a customizable database service are not possible in Auth0 data store. For example, if you wanted to make a specific query like `SELECT users.favGenre, access.roles FROM users, access WHERE users.user_id = access.user_id`. This is something you can't do with in the auth0 data store. Using a separate database will allow you to manage your data however you see fit.
+
+### Example
+
+Here is an example of data that is associated with a user but not with authenticating that user in the app: In the case of our mobile music application, the user’s music needs to be saved, so they can find it easily when they log in again. This data is not required in the process of authenticating the user for the app, but favorite songs and artists are personal data that should be associated with the user. So we would want to store this data in a separate database connected to the backend of our mobile app, instead of in the Auth0 data store. Here is how we did this:
```

Comment:
@eharkins This is the first time this "mobile music application" is mentioned in the doc. When I read the sentence "In the case of our mobile music application" it made me feel like I missed something earlier. Rephrase maybe?

Also, I don't think the ":" after the first sentence looks good, I would replace with "."